migrations/20210805154736_alter_view_transactions_add_signer.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE view_transactions
+    ADD signers JSONB NOT NULL;
```

Comment:
For existing transactions in the table, will `NOT NULL` fails them? Perhaps put an empty array as default value ?

Reply:
Running in testnet env has no issue on this. But true, more safety to put a default value here. Will do it.